src/main/java/seedu/address/logic/commands/UsageMessage.java

```diff
@@ -0,0 +1,256 @@
+                        MAPPING_DELETE.getValue(),
+                        MAPPING_SEARCH.getValue(),
+                        MAPPING_SORT.getValue())),
+    NOTE_ADD(
+        NoteCommand.COMMAND_WORD,
+        NoteAddCommand.ACTION_WORD,
+        "[content] [tags]",
+        "Adds a note"),
+    NOTE_DELETE(
+        NoteCommand.COMMAND_WORD,
+        NoteDeleteCommand.ACTION_WORD,
+        "[index]",
+        "Deletes a note"),
+    NOTE_LIST(
+        NoteCommand.COMMAND_WORD,
+        NoteListCommand.ACTION_WORD,
+        "",
+        "Lists all notes"),
+    NOTE_SEARCH(
+        NoteCommand.COMMAND_WORD,
+        NoteSearchCommand.ACTION_WORD,
+        "[query]",
+        "Searches for notes with query is found in the note tag"),
+    NOTE_TAG(
+        NoteCommand.COMMAND_WORD,
+        NoteTagCommand.ACTION_WORD,
+        "[index] [tag]",
+        "Add a tag to a note"),
+    NOTE_UPDATE(
+        NoteCommand.COMMAND_WORD,
+        NoteUpdateCommand.ACTION_WORD,
+        "[index] [content]",
+        "Updates a note to the specified content"),
+    NOTE_CLEAR_TAG(
+        NoteCommand.COMMAND_WORD,
+        NoteClearTagCommand.ACTION_WORD,
+        "[index]",
+        "Removes all tags to a note"),
+    NOTE(String.join("\n",
+                     "Note Commands:\n",
+                     NOTE_ADD.getValue(),
+                     NOTE_CLEAR_TAG.getValue(),
+                     NOTE_DELETE.getValue(),
+                     NOTE_LIST.getValue(),
+                     NOTE_SEARCH.getValue(),
+                     NOTE_TAG.getValue(),
+                     NOTE_UPDATE.getValue())),
+    PARTNERCOURSE_ADD(
+        PartnerCourseCommand.COMMAND_WORD,
+        PartnerCourseAddCommand.ACTION_WORD,
+        "[university] [partnercode] [partnername] [partnerunit] [partnerdescription]",
+        "Adds a partner course"),
+    PARTNERCOURSE_DELETE(
+        PartnerCourseCommand.COMMAND_WORD,
+        PartnerCourseDeleteCommand.ACTION_WORD,
+        "[partnercode]",
+        "Deletes a partner course"),
+    PARTNERCOURSE_LIST(
+        PartnerCourseCommand.COMMAND_WORD,
+        PartnerCourseListCommand.ACTION_WORD,
+        "",
+        "Lists all partner courses"),
+    PARTNERCOURSE_SEARCH(
+        PartnerCourseCommand.COMMAND_WORD,
+        PartnerCourseSearchCommand.ACTION_WORD,
+        "[attribute] [query]",
+        "Searches for a partner course where query is found in the selected attribute\nAttributes:\n"
+            + "partnercode\npartnername\nuniversity"),
+    PARTNERCOURSE_SORT(
+        PartnerCourseCommand.COMMAND_WORD,
+        PartnerCourseSortCommand.ACTION_WORD,
+        "[attribute]",
+        "Sorts partner courses by the selected attribute\nAttributes:\n"
+            + "partnercode\npartnername\nuniversity"),
+    PARTNERCOURSE_UPDATE(
+        PartnerCourseCommand.COMMAND_WORD,
+        PartnerCourseUpdateCommand.ACTION_WORD,
+        "[partnercode] [attribute] [newvalue]",
```

Comment:
need to include university! so [university] [partnercode] [attribute]  [updatedValue]